x-pack/plugins/saved_objects_tagging/public/management/components/_action_bar.scss

```diff
@@ -0,0 +1,11 @@
+.tagMgt__actionBar {
+  > .euiHorizontalRule {
+    margin: 8px 0 3px;
+  }
+}
```

Comment:
Quite similar, the expected design requires the horizontal line separator to be very close to the table. I could have used a custom component instead of `EuiHorizontalRule` with hacked margin, however it looked like a worse solution that just changing a margin on the already fully functional component.

<img width="1208" alt="Screenshot 2020-11-09 at 10 01 44" src="https://user-images.githubusercontent.com/1532934/98520633-9c5eb280-2272-11eb-8d1b-c80b26caf80a.png">

Reply:
What you're creating is simply a border to your action bar. Instead of using the component, use styles like:

```scss
.tagMgt__actionBar {
  border-bottom: $euiBorderThin;
  padding-bottom: $euiSizeS;
  margin-bottom: $euiSizeXS;
}
```